redfish-core/lib/log_services.hpp

```diff
@@ -1011,8 +1011,8 @@ inline void createDumpTaskCallback(
                 "Location: " + dumpEntryPath + http_helpers::urlEncode(dumpId);
             taskData->payload->httpHeaders.emplace_back(std::move(headerLoc));
 
-            BMCWEB_LOG_DEBUG << createdObjPath.str
-                             << ": Dump creation task completed";
+            BMCWEB_LOG_CRITICAL << "INFO: " << createdObjPath.str
```

Comment:
Why do we need to print this everytime?

Reply:
this is need to confirm if dump creation completed

Reply:
only one trace for each dump created

Reply:
What the PR message describes and what this PR is doing seem different

Reply:
currently there is no trace from bmcweb to confirm dump creation completed. this is really helpful to confirm if ther is a particular dump creation completed from bmcweb side